src/security.tf

```diff
@@ -7,12 +7,16 @@ resource "azurerm_resource_group" "sec_rg" {
 
 
 module "key_vault" {
-  source              = "git::https://github.com/pagopa/azurerm.git//key_vault?ref=v1.0.48"
+  source              = "git::https://github.com/pagopa/azurerm.git//key_vault?ref=add-logging-kv"
```

Comment:
This link should reference an officially released version of the module. cc @pasqualedevita